packages/cli/.prettierignore

```diff
@@ -0,0 +1,13 @@
+dist
```

Comment:
Does monorepos need to have .prettierignore in each package?
isn't it global ?

Reply:
yes. `.prettierignore` is per repo